### DISCUSSION TITLE
Keyboard Shortcuts

### DISCUSSION THREAD

**Discussion by zpsmi** [evilsocket/opensnitch#727] [Ideas]
The options to change duration, allow or deny connections using keyboard would be great.

Pressing a key to change duration and others for allowing and denying connections would prevent going back and forth between keyboard and mouse.

I am switching to Linux from Mac and use Little Snitch on Mac side.

**Comment by sulfurcatwalk** [comment]
Seconded. Also give command line option to allow/deny a rule. This would make it easier to stop connections for certain sites/apps on certain connections once bound to a keybinding.
